qa/sc_evm_feeHistory.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
+import json
+import logging
+from decimal import Decimal
+
+from SidechainTestFramework.account.ac_chain_setup import AccountChainSetup
+from SidechainTestFramework.account.ac_utils import ac_makeForgerStake
+from SidechainTestFramework.account.httpCalls.transaction.createEIP1559Transaction import createEIP1559Transaction
+from SidechainTestFramework.account.utils import convertZenToZennies
+from SidechainTestFramework.scutil import generate_next_block, \
+    assert_equal
+from test_framework.util import forward_transfer_to_sidechain, fail
+
+"""
+Check eth_feeHistory pending block tag works properly
+
+Configuration:
+    - 2 SC nodes connected with each other
+    - 1 MC node
+
+Test:
+    - Add some transactions to the SC nodes. 
+    - Get node 1 and node 2 fee histories with pending block tag
+    - Verify that they are equal and return results with valid lengths w.r.t. blocks mined and transactions
+    - Get node 1 and node 2 fee histories with latest block tag
+    - Verify that they are equal and return results with valid lengths w.r.t. blocks mined and transactions
+"""
+
+
+class SCEvmFeeHistory(AccountChainSetup):
```

Comment:
This test is OK. It should be updated also EthServiceTest.scala with a test on "pending" tag.

Reply:
Done